package.json

```diff
@@ -20,106 +20,105 @@
+        "@types/loadable__component": "^5.13.4",
+        "@types/node": "^16.11.9",
+        "@types/react": "^17.0.35",
         "@types/react-bootstrap": "^1.0.1",
-        "@types/react-dom": "^16.9.10",
-        "@types/react-router-dom": "^5.1.7",
-        "@types/react-transition-group": "^4.4.0",
+        "@types/react-dom": "^16.9.14",
+        "@types/react-router-dom": "^5.3.2",
+        "@types/react-transition-group": "^4.4.4",
         "@types/webpack": "^5.28.0",
         "@typescript-eslint/eslint-plugin": "^3.10.1",
         "@typescript-eslint/parser": "^3.10.1",
-        "autoprefixer": "^10.2.5",
+        "autoprefixer": "^10.4.0",
+        "babel-loader": "^8.2.3",
         "copy-webpack-plugin": "^8.1.1",
-        "css-loader": "^5.2.5",
-        "eslint": "^7.19.0",
-        "eslint-config-prettier": "^6.15.0",
+        "css-loader": "^5.2.7",
+        "eslint": "^7.32.0",
         "eslint-plugin-import": "^2.22.1",
-        "eslint-plugin-prettier": "^3.3.1",
-        "eslint-plugin-react": "^7.22.0",
+        "eslint-plugin-radar": "^0.2.1",
+        "eslint-plugin-react": "^7.27.1",
         "eslint-plugin-simple-import-sort": "^5.0.3",
-        "fast-sass-loader": "^2.0.0",
+        "fast-sass-loader": "^2.0.1",
+        "file-loader": "^6.2.0",
         "follow-redirects": "^1.13.2",
-        "fork-ts-checker-webpack-plugin": "^6.2.10",
-        "html-webpack-plugin": "^5.3.1",
+        "fork-ts-checker-webpack-plugin": "^6.4.0",
+        "html-webpack-plugin": "^5.5.0",
+        "patch-package": "^6.4.7",
         "postcss": "^8.2.15",
         "postcss-loader": "^5.3.0",
-        "prettier": "^2.2.1",
-        "react-dom": "^17.0.1",
-        "react-refresh": "^0.10.0",
-        "sass": "^1.32.13",
+        "react-dom": "^17.0.2",
+        "react-refresh": "^0.11.0",
+        "sass": "^1.43.4",
         "source-map-loader": "^2.0.2",
         "style-loader": "^2.0.0",
         "svg-loader": "0.0.2",
+        "swagger-typescript-api": "9.3.1",
         "tiny-typed-emitter": "^2.0.3",
-        "typescript": "^4.4.3",
-        "webpack": "^5.37.1",
-        "webpack-cli": "^4.7.0",
-        "webpack-dev-server": "^4.0.0-beta.3"
+        "typescript": "^4.5.2",
+        "webpack": "^5.64.2",
+        "webpack-cli": "^4.9.1",
+        "webpack-dev-server": "^4.5.0"
+    },
+    "scripts": {
+        "start": "yarn webpack serve --mode development",
+        "build": "yarn webpack --mode production",
+        "msbuild": "node scripts/msbuild.js && yarn run build",
+        "build-dev": "yarn webpack --mode development",
+        "build-github": "yarn webpack --mode production --git 1",
+        "lint": "eslint --max-warnings=0 \"src/**\"",
+        "lint-fix": "eslint --fix --max-warnings=0 \"src/**\"",
+        "test": "yarn run lint && yarn run build",
+        "postinstall": "yarn run generate-api",
+        "generate-api": "yarn run patch && node scripts/api_build.js",
+        "patch": "yarn patch-package"
```

Comment:
You broke like half of these, `yarn webpack` isn't a command, use npx or something

Reply:
it is on my side (and the ci), dunno about yours though

Reply:
Dunno why it didn't work previously, ma bad